Automattic/jetpack
# Use inline fill for form variation colours

Re-introduce icon colors for Jetpack form variations:

![Screenshot 2020-09-08 at 10 35 32](https://user-images.githubusercontent.com/411945/92453284-5e380a80-f1bf-11ea-8fd2-ce02b7700735.png)

Relates to: #17089 #17092 

#### Changes proposed in this Pull Request:

* None, this reverts a revert

#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



N/A

#### Testing instructions:






* Pull the branch
* Head to the Gutenberg editor
* Check the icon colours for all Jetpack blocks are green

#### Proposed changelog entry for your changes:

* N/A
